pkgs/top-level/python-packages.nix

```diff
@@ -7490,6 +7513,33 @@ in modules // {
     };
   };
 
+  nxt_python = buildPythonPackage rec {
```

Comment:
done in https://github.com/NixOS/nixpkgs/commit/5b3331ed2c8f68c0a07ee81fdeeab0abb2b2c0a6